src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partnerType = person.sex === 'm' ? 'wife' : 'husband';
+  const personPartner = person.isMarried
+    ? `${person.partnerName} is my ${partnerType}`
+    : 'I am not married';
+
+  return (
+    <div>
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
Use destructuring for `person`
```suggestion
      <h2 className="Person__name">{`My name is ${name}`}</h2>
```